controllers/seller/getSellerFromStore.js

```diff
@@ -0,0 +1,24 @@
+const Seller = require('../../models/seller');
+const Store = require('../../models/store');
+
+const getSellerFromStore = async (req, res) => {
+	try {
+		const { storeId } = req.params;
+		console.log('StoreID?', storeId);
+
+		const store = await Store.findOne({ _id: storeId });
+		console.log('Is there a store', store);
+
+		const sellerId = store.seller;
+		const seller = await Seller.findOne({ _id: sellerId });
+		console.log('Seller?', seller);
```

Comment:
Remove this console log if it's not needed

Reply:
Should be up to date now.